src/test/regress/expected/scale_factor.out

```diff
@@ -1,33 +1,68 @@
 -- start_ignore
+create or replace function get_explain_xml_output(query_string text)
```

Comment:
ignore content in output is redundant

Reply:
Done

Reply:
start/end _ignore words should not be deleted from output

Reply:
What did you mean saying that ignored content is redundant?

Reply:
you may cut all between start_ignore and end_ignore in output

Reply:
Done